lib/hermetic/youtrack.ex

```diff
@@ -21,34 +21,92 @@ defmodule Hermetic.YouTrack do
   end
 
   @doc """
-    Return URL to YouTrack avatar for the given username.
+  Send HTTP PUT request to provided YouTrack endpoint.
+  """
+  def put!(endpoint) do
+    headers = [
+      {"authorization", "Bearer " <> token()},
+    ]
+
+    HTTPoison.put!(base_url() <> endpoint, "", headers)
+  end
+
+  @doc """
+  Send HTTP POST request to provided YouTrack endpoint.
+  """
+  def post!(endpoint) do
+    headers = [
+      {"authorization", "Bearer " <> token()},
+    ]
+
+    HTTPoison.post!(base_url() <> endpoint, "", headers)
+  end
+
+  @doc """
+  Create a new issue and return the issue id
+  """
+  @spec create_issue(String.t(), String.t(), String.t()) :: String.t()
+  def create_issue(project, summary, description) do
+    resp = put!("/rest/issue?" <> URI.encode_query([
```

Comment:
Instead of explicit `URI.encode_query/1` and concatenation, use HTTPoison params arg instead (see https://github.com/edgurgel/httpoison/pull/37 for usage example).